source/preferences.cpp

```diff
@@ -231,6 +231,16 @@ static void loadXMLSetting(float * var, const char * name)
 			*var = atof(tmp);
 	}
 }
+static void loadXMLSetting(uint32 * var, const char * name)
```

Comment:
Why do we need this method?

Reply:
This is something I put in to add the SuperOverclockMultiplier and OverclockCycle settings in the Snes9x menu. At the moment is has not use so I will remove it.